### ISSUE TITLE
Introducing the Morea Framework 

### ISSUE THREAD

**Issue by philipmjohnson** [education/teachers#24]
Greetings, all,

I would like to let you know about a mashup of GitHub, Jekyll, Bootstrap, and a tiny Ruby plugin that I've created to facilitate the design and construction of courses. It's called the Morea Framework:

http://morea-framework.github.io/

It implements a "pedagogical pattern" involving Modules, Learning Outcomes, Readings, Experiences, and Assessments that (at least for me) makes it easier to express the intent of my courses: when, where, why, and how to learn.  (It also makes it easier to know when I've failed to express this intent.) 

Morea is in early days; there are currently about a half dozen classes at the University of Hawaii using it, and I will be using it exclusively for the foreseeable future. :)

The home page has a six minute video that provides some details on why I created the framework, and why (or why not) you might be interested in trying it out. 

Please check it out if you like, and feel free to email me or reply to this thread if you have questions or comments. 

Philip

**Comment by crguezl** [comment]
Relacionado con esto:

Morea Framework:

* [Easily generate educational websites organized by Modules, Outcomes, Readings, Experiences, and Assessments](http://morea-framework.github.io/)

**Comment by Badlapje** [comment]
This looks like a nice framework, nice job @philipmjohnson .  Given it's been a couple years since this was posted i've a couple questions: 
- are you still using this and what's the experience been so far?
- what's the technical barrier like for students when they use this?  
- Is this usable for non CS students (eg. could this be used by eg. Psychology teachers and their students)?
- I haven't gotten round to testing this yet, as i just stumbled over it while checking out github education resources.  But is this open source?  I ask because we're looking to develop our own learning tools and being able to amend morea would be very interesting.

**Comment by philipmjohnson** [comment]
Yes, @Badlapje, the Morea Framework is still in active use every semester. We haven't released any new updates to the framework recently because, well, there hasn't been any pressing need. Here's my [software engineering course site for this semester](http://courses.ics.hawaii.edu/ics314f16/), for example.

The technical barrier is low/non-existent for students. I did surveys for a few semesters and the majority of students felt it was easier to use than the alternative learning management system at UH (Sekai). I don't think the degree program of the students matters at all.

The technical barrier is significantly higher for teachers; you need to know a little bit about Jekyll, a little bit about GitHub, a little bit about Markdown, and a little bit about Bootstrap. And, you need to buy in at least a little bit to the idea that organizing your course into modules, outcomes, readings, experiences, and assessments. 

Yes, it's open source, and I am happy to be a resource person if you want to use it as a basis for your own efforts.
